indra/newview/llspatialpartition.cpp

```diff
@@ -847,6 +847,43 @@ void LLSpatialGroup::handleChildAddition(const OctreeNode* parent, OctreeNode* c
 	assert_states_valid(this);
 }
 
+//virtual
+void LLSpatialGroup::rebound()
```

Comment:
Are you sure this is the right way? I think our common solution for such cases is to keep parent visible if any of it's children are visible, may be that check needs to be fixed to account avatar's extents properly? How often does rebound happen?